app/cells/select/show.slim

```diff
@@ -0,0 +1,9 @@
+.form-group class=(errors.any? ? 'has-error' : '')
+  - unless options[:skip_label]
+    label.control-label for=id
+      = label
+  = hidden_field_tag name
+  = react_component('ui_components.Select', react_options)
+  - if errors.any?
+    span.help-block
+      = errors.join('<br />')
```

Comment:
Not that I thought about it, but...
```ruby
simple_format(errors.join("\n"))
```
doesn't really look better, does it?

On the other hand, I just now realize that I should probably add a `.html_safe`. So, maybe `simple_format` does look better after all. ;)